esp8266/modules/webrepl_setup.py

```diff
@@ -1,83 +1,95 @@
 import sys
-import socket
-import time
-
-from websocket import *
-import websocket_helper
-
-
-def setup_server():
-    s = socket.socket()
-    s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-
-    ai = socket.getaddrinfo("0.0.0.0", 8266)
-    addr = ai[0][4]
-
-    s.bind(addr)
-    s.listen(1)
-    return s
-
-def getpass(stream, prompt):
-    stream.write(prompt)
-    passwd = b""
+#import uos as os
+import os
+
+RC = "./boot.py"
+CONFIG = "./webrepl_cfg.py"
+
+def exists(fname):
+    try:
+        with open(fname):
+            pass
+        return True
+    except OSError:
+        return False
+
+def get_status():
+    with open(RC) as f:
+        for l in f:
+            if "webrepl" in l:
+                if l.startswith("#"):
```

Comment:
What if the '#' is in the second column? A much easier check would be to simply look for the webrepl module in the memory.